src/main/java/net/sf/jabref/logic/layout/LayoutEntry.java

```diff
@@ -644,9 +646,14 @@ private LayoutFormatter getLayoutFormatterByName(String name) throws Exception {
                             int startParam = i;
                             i++;
                             boolean escaped = false;
-                            while (((i + 1) < c.length) && !(!escaped && (c[i] == '"') && (c[i + 1] == ')'))) {
+                            while (((i + 1) < c.length)
```

Comment:
When I remeber it correclty, there is also a method in StringUtil which does the same and checks for curly(?) braces...